styles.css

```diff
@@ -0,0 +1,90 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+
+}
+.app {
+    width: 100vw;
+    height: 100vh;
+    background: url(images/background.png);
+    background-repeat: no-repeat;
+    background-size: cover;
```

Comment:
- [ ] Please add `background-position:center` to get the backdrop image centralized
> The width and height are just OK. only add the background position